index.html

```diff
@@ -5838,6 +5838,20 @@ <h3>Group Control</h3>
         </p>
       </section>
     </section>
+    <section>
+      <h2>Changing the DID subject</h2>
+      <p>
+A <a>DID document</a> has exactly one <a>DID subject</a>, identified by exactly
+one <a>DID</a>, expressed as the value of the the <code><a>id</a></code>
+property. This property value is immutable for the lifetime of the
+<a>DID document</a>.
+      </p>
+      <p>
+However, it is possible that the resource <em>identified</em> by the <a>DID</a>
+may change over time. This is under the exclusive authority of the
+<a>DID controller</a>. For more details, see section <a href="#persistence"></a>.
```

Comment:
I think this follow-on paragraph makes it clear that the DID subject may change over time. But editorially, we could add this to clarify:

```suggestion
However, it is possible that the resource <em>identified</em> by the <a>DID</a>,
the <a>DID subject</a>, may change over time. This is under the exclusive authority
of the <a>DID controller</a>. For more details, see section <a href="#persistence"></a>.
```